src/apis/setHeaderItems.js

```diff
@@ -74,7 +75,7 @@ export default store => callback => {
 /**
  * A class which contains header APIs.<br/><br/>
  * <span style="color: red; font-size: 1.2em; font-weight: bold">⚠</span> You must NOT instantiate this yourself. Access the header instance in {@link WebViewer#setHeaderItems setHeaderItems} as follows:
- * @name WebViewer.Header
+ * @name Header
```

Comment:
Hmm if we do this to `Header` as well, then what's the way to find the documentation for it? Currently it's here https://www.pdftron.com/api/web/WebViewer.html#toc2__anchor.

Reply:
It will be in the "classes" menu and will be linked to from appropriate methods